packages/api/src/routes/nfts-upload.js

```diff
@@ -45,19 +45,27 @@ export async function upload(event, ctx) {
     }
     nftSize = size
   } else {
-    const blob = await event.request.blob()
+    let blob = await event.request.blob()
     if (blob.size === 0) {
       throw new HTTPError('Empty payload', 400)
     }
-    const { cid, size } = await cluster.add(blob)
+    if (
+      contentType.includes('application/car') &&
+      blob.type !== 'application/car'
+    ) {
+      // force the content type on the blob without duplicating content
+      blob = blob.slice(0, blob.size, 'application/car')
+    }
```

Comment:
This may not strictly be needed, it is a left over from when I was struggling to get the content-type:application/car to show up in the worker when i set it on the upload... I'm not clear on if the blob as provided in the worker context is guaranteed to always have it's content-type set to match the content-type header on the POST that got it there.

Reply:
...and it's vital for us that it does here as it's the only flag to the cluster client to treat the add as a CAR add.